test-apps/ui-test-app/src/backend/main.ts

```diff
@@ -31,8 +31,8 @@ import { ECSchemaRpcImpl } from "@itwin/ecschema-rpcinterface-impl";
     initializeLogging();
 
     const iModelHost = new IModelHostConfiguration();
-    const iModelClient = new  IModelsClient({ api: { baseUrl: `https://${process.env.IMJS_URL_PREFIX ?? ""}api.bentley.com/imodels`}});
-    iModelHost.hubAccess = new BackendIModelsAccess(iModelClient);
+    const iModelClient = new IModelsClient({ api: { baseUrl: `https://${process.env.IMJS_URL_PREFIX ?? ""}api.bentley.com/imodels` } });
+    iModelHost.hubAccess = new BackendIModelsAccess(iModelClient) as any;
```

Comment:
@calebmshafer it is a bit tricky (especially since we are making some breaking changes to @internal and @beta types), since the `@itwin/imodels-access-backend` depends on a platform interface, but the test apps and integration tests depends on the `@itwin/imodels-access-backend`..